docs/use/basic.md

```diff
@@ -0,0 +1,98 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: '0.8'
+    jupytext_version: '1.4.1'
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# An example Jupyter Notebook
+
+This notebook is a demonstration of directly-parsing Jupyter Notebooks into
+Sphinx using the MyST parser.[^download]
+
+[^download]: This notebook can be downloaded as
+            **{jupyter-download:notebook}`basic`** and {download}`basic.md`
```

Comment:
Any particular reason to use `jupyter-download` here and not just `download`?

Reply:
Because the file is taken from the `jupyter_execute` folder, i.e. the post-execution notebook, not the source folder as `download` looks at. I guess see `jupyter-sphinx` for the actual logic.

Reply:
right - I remember now, thanks